config/jobs/image-pushing/k8s-staging-autoscaling.yaml

```diff
@@ -21,3 +21,25 @@ postsubmits:
               - --scratch-bucket=gs://k8s-staging-autoscaling-gcb
               - --env-passthrough=PULL_BASE_REF
               - addon-resizer
+    - name: post-autoscaler-push-vpa-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-autoscaling-vpa-images, sig-k8s-infra-gcb
+      decorate: true
+      branches:
+        - ^master$
```

Comment:
this doesn't need to be a vpa branch similar to addon-resizer?

(related: we should probably use a regex more like `^addon-resizer-release-` instead of pinning to only build a particular release branch, these jobs have limited approvers by necessity)

Reply:
addon-resizer is a bit of an anomaly, the main development happens in a separate branch, we need to clean that up :/

Thanks Ben!

Reply:
It appears to me that it happens on multiple release versioned branches? The current regex only matches one of them, if we matched all of them then we wouldn't have to update it here.

Reply:
Took a look, we actually ONLY use the 1.8 branch for development. The others are no longer used at all so we actually don't want to build them. The way we have it here is correct.

Thanks for noticing this though.

Reply:
ACK, ... but is the plan to use 1.8 branch forever??

Reply:
anyhow, not actually related to this change

Reply:
for now, yes... With no plans to change that in the near future.